bakerytest/bakerytest.go

```diff
@@ -73,3 +73,8 @@ func (d *Discharger) PublicKeyForLocation(loc string) (*bakery.PublicKey, error)
 	}
 	return nil, bakery.ErrNotFound
 }
+
+// URL returns the URL the discharger is using.
```

Comment:
I don't think this function is necessary - we can use d.Service.Location() instead (perhaps we should document that on the Discharger type).